msi/rust.wxs

```diff
@@ -149,6 +154,10 @@
         <!-- Specifies a single cab file to be embedded in the installer's .msi. -->
         <MediaTemplate EmbedCab="yes" CompressionLevel="high" />
 
+        <!-- Send a WM_SETTINGCHANGE message to tell processes like explorer to update their
+             environments so any new command prompts get the updated %PATH% -->
+        <CustomActionRef Id="WixBroadcastEnvironmentChange" />
```

Comment:
Are you sure this changes anything?   The [WriteEnvironmentStrings](https://msdn.microsoft.com/en-us/library/aa372883.aspx) action already sends  `WM_SETTINGCHANGE` message.
This doesn't affect the already open command prompts, though, because they aren't listening for this message.